src/libs/API.js

```diff
@@ -530,11 +530,11 @@ function Report_UpdateLastRead(parameters) {
 
 /**
  * @param {Object} parameters
- * @param {Number} parameters.email
+ * @param {String} parameters.email
  * @returns {Promise}
  */
 function ResendValidateCode(parameters) {
-    const commandName = 'ResendValidateCode';
+    const commandName = 'SetCashPassword';
```

Comment:
Are we changing the API method name too since we are changing the `commandName`? 🤔

Reply:
Yes! I'm actually going to change it back to ResendValidateCode today, and modify `ResendValidateCode` in Auth to send out the right notification.
```suggestion
    const commandName = 'ResendValidateCode';
```